docs/overview/index.md

```diff
@@ -9,64 +9,120 @@ myst:
 
-(overview-6-label)=
+- **Plone is extensible.**
+  There is a multitude of add-on products for Plone to add new features and content types.
+  In addition, Plone can be scripted using web standard solutions and open source languages.
 
-### Headless CMS with JavaScript frontend through a REST API
+- **Plone is technology neutral.**
+  Plone can interoperate with most relational database systems—both open source and commercial—and runs on a vast array of platforms, including Linux, Windows, macOS, and BSD.
 
 
-(overview-7-label)=
 
-### Explain frontend-backend terms
+(overview-4-label)=
 
+## High Level Overview for Developers
 
-(overview-8-label)=
+Plone is a content management platform with its backend written in Python.
+It is built on top of the open-source Zope web application server and development system. 
+Plone is making use of the pluggable Zope Component Architecture (ZCA) to provide a highly modular and extensible system.
+Up until version 6, Plone has used server-side rendering to generate HTML-based content, with advanced resource management features for adding and bundling CSS and JavaScript. 
+Additionally, Plone's use of the ZCA makes it easy to extend and custosize, allowing users to create unique, feature-rich websites that are tailored to their specific needs.
 
-### Security
+With the release of Plone 6, you now have the option to choose from two different out-of-the-box supported configurations when setting up a new Plone website.
+The Python-based backend server in Plone alone can still be used to render content server-side and deliver HTML to the browser, a setup that is referred to in the Plone documentation as "Classic UI". 
+This configuration has been supported by Plone since its initial release and is still available in the latest versions of the platform. 
+For container-based deployment, only the plone-backend image is required, or a derivative image with customisations added. 
 
+The default and advised configuration for new websites in Plone is to use our new React Based javascript frontend called 'Volto'.
+For this setup you still need to run the Python based backend server, but with the REST API enabled and an updated configuration profile.
+In addition a separate NodeJS based frontend server will serve the javascript frontend resources and provide SSR with hydration.
+To deploy this setup using containers you will need the plone-frontend image of the frontend server.
 
-(overview-9-label)=
+Plone 6 is the version Plone where we support two programming language stacks, one for Python and Javascript.
+The documentation has been rewritten, but for this first release you will find some repetition of concepts in the documentation structure.
+For example for the development setup and information on deployment options.
+It will take some time before we find and can implement the best structure to explain these new possibilities and expansion of Plone its capabilities.
```

Comment:
```suggestion
It will take some time until we find and can implement the best structure to explain these new possibilities and the expansion of Plone's capabilities.
```